src/parser/mod.rs

```diff
@@ -4217,7 +4217,7 @@ impl<'a> Parser<'a> {
         }
     }
 
-    fn parse<T: FromStr>(s: String, loc: Location) -> Result<T, ParserError>
+    pub fn parse<T: FromStr>(s: String, loc: Location) -> Result<T, ParserError>
```

Comment:
```suggestion
    fn parse<T: FromStr>(s: String, loc: Location) -> Result<T, ParserError>
```
was there a requirement for this change to pub? if needed elsewhere inthe crate we can use `pub(crate)``instead